src/Components/Milestones/Assessment.css

```diff
@@ -17,7 +17,9 @@
     margin-bottom: 20px;
 }
 
```

Comment:
idk which style is related to this, but the bars under each of the labels have uneven spacing w the below content: in the sections without checkboxes (client info, education, dream job) it could use some extra padding

Reply:
added some extra padding! Let me know if I missed something!